### ISSUE TITLE
Broken link for benchmark results

### ISSUE THREAD

**Issue by garry-cairns** [open-telemetry/opentelemetry-python-contrib#1556]
The link provided in https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/docs/performance/benchmarks.rst for benchmark results is a 404.

There is an open issue related to benchmarks at https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1016 but it doesn't look related to me.

I've looked in [recent actions](https://github.com/open-telemetry/opentelemetry-python-contrib/actions/workflows/test.yml) and nothing is jumping out at me as being the obvious place where benchmarks are running. Do these still run by default?

**Comment by srikanthccv** [comment]
No, they don't run anymore. We have temporarily disabled them as they were flaky and didn't highlight any real performance issues.

**Comment by bmbouter** [comment]
I came to report this issue, but then I found this. Is there some way that we can see what data was there at least before it was broken?

**Comment by garry-cairns** [comment]
@bmbouter I didn't find that so in the end I just conducted my own test by starting n spans of different weights and timing/sizing them in my application.

**Comment by bmbouter** [comment]
Thanks for the info! What do you mean by "different weights"? Like the amount of data/attributes stored in them or?
